src/app/components/pop-up-how-to-play/pop-up-how-to-play.component.ts

```diff
@@ -1,24 +1,23 @@
 import { Component } from '@angular/core';
-import { PopUpHowToPlayService } from 'src/app/services/pop-up-how-to-play/pop-up-how-to-play.service';
+import { MatDialogRef } from '@angular/material/dialog';
 
 @Component({
   selector: 'app-pop-up-how-to-play',
   templateUrl: './pop-up-how-to-play.component.html',
   styleUrls: ['./pop-up-how-to-play.component.scss']
 })
 export class PopUpHowToPlayComponent {
+  constructor(public dialogRef: MatDialogRef<PopUpHowToPlayComponent>) {}
 
-  constructor(public myService: PopUpHowToPlayService) {}
-
-  mudarDeslocamento(): void {
-    const novoDeslocamento = this.myService.deslocamento + 450; // Ajuste conforme necessário
-    this.myService.alterarDeslocamento(novoDeslocamento);
+  /**
+   * Função responsável por fechar o modal quando o botão "cancel" for acionado.
+   */
+  public closeModal(): void {
+    this.dialogRef.close();
   }
-
-  handleKeyDown(event: KeyboardEvent): void {
-    if (event.key === 'Enter' || event.key === 'Space') {
-      this.mudarDeslocamento();
+  public handleKeyDown(event: KeyboardEvent): void {
```

Comment:
- Colocar uma linha em branco em cima do método
- Documentar o método

Reply:
Tentei inserir as linhas em branco entre o métodos mas o Lint não está permitindo.

Trailing spaces not allowed.eslint[no-trailing-spaces](https://eslint.org/docs/latest/rules/no-trailing-spaces)
Delete ··eslint[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)

Reply:
Provavelmente ele reclamou do espaço no começo da linha e não da linha em si...